src/client/WebhookClient.js

```diff
@@ -10,17 +11,48 @@ const Webhook = require('../structures/Webhook');
  */
 class WebhookClient extends BaseClient {
   /**
-   * @param {Snowflake} id The webhook's id
-   * @param {string} token Token of the webhook
+   * The data for the webhook client containing either an id and token or just a URL
+   * @typedef {Object} WebhookClientData
+   * @property {(WebhookClientDataIdWithToken|WebhookClientDataURL)} data The data for the webhook client
+   */
```

Comment:
this typedef is
```ts
{
  data: {
    id: string;
    token: string;
  } | {
    url: string;
  }
}
```
convert the typedefs below into a single typedef, and use that instead
(the jsdoc typedefs aren't usually handled this way, the typings are left for the more specific unions)